array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,22 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function showPeopleBelongIn(array, houseName) {
+  const filteredArray = array.filter((person) => person.house === houseName);
+  filteredArray.forEach((person) => {
+    const {firstName, lastName} = person;
+    console.log(`${firstName} ${lastName}`);
+  })
+}
+
+function showTeachersWithPets(array) {
+  const filteredArray = array.filter((person) => person.pet !== null && person.occupation === "Teacher");
+  filteredArray.forEach((person) => {
+    const {firstName, lastName} = person
+    console.log(`\n${firstName} ${lastName}`);
+  })
+}
+
+showPeopleBelongIn(hogwarts, "Gryffindor");
+showTeachersWithPets(hogwarts);
```

Comment:
![image](https://github.com/CodeYourFuture/Module-JS3/assets/25291974/793c639a-68ad-40f5-9e27-357622205d4b)

You can get rid of these red signs by adding an empty new line to the end of your files.
I think there's a bit of history why it's commonly done, but also a genuine technical reason, so I think it's a good habit to get into in general.

Reply:
Empty line added.
Thanks.